proposals/hackygolucky-EXPECTATIONS/EXPECTATIONS.md

```diff
@@ -0,0 +1,108 @@
+
+### Communicated expectations
+
+##### Expectations of the Foundation
+- Protect Intellectual Property (IP) including trademarks 
+    - Enable the project communities to use the trademarks 
+    - Educate members, new and old, about proper use of the projects' IP 
+
+- Provide Legal Advice and Legal Best Practices 
+    - Provide legal resources and representation that support the needs of the project 
+    - Create a well-defined communications path for project legal needs 
+
+- Software / Hardware resource facilitation (for use by the project) 
+    - Examples: 
+        - OKTA Identity Service for Individual Membership 
+        - Headless CMS for website 
+        - Translation and Internationalization Platform 
+        - Have asked for approval to buy mac minis….  
+
+- Provide Ongoing Supporting Personnel to help manage community and technical efforts 
+    - Examples: 
+        - Community Manager 
+        - Education Manager 
+
+- Provide a pathway for community personnel requests for resources including but not limited to Employee, Contractor, Committed Volunteer, and Outsourced roles 
+    - Examples: 
+        - Designer for a project website 
+        - Site Reliability Engineer (SRE) to monitor uptime and service quality of Build, Website, and Automation Infrastructure 
+
+- Grow Foundation Membership and Expand the Network of folks engaged with the projects (and facilitate contribution from the Members to the projects) 
+    - Provide compelling membership value propositions 
+    - Recruit new members 
+    - Enable members to meet regularly 
+
+- Foundation representatives should regularly attend top-level meetings 
+    - When needed, the Executive Director will work with the top-level committees to ensure their role as a liaison is fulfilled. 
+
+- Engage members of the top-level committees inside of Foundation functions, like the Marketing Committee, Legal Committee, and Events meetings. 
+- Increase the adoption of projects across individuals and companies by 
+    - Promoting initiatives by the top-level committees 
+    - Financing initiatives (as per some agreed level of priority) that have this objective. This is related to community personnel requests outlined earlier 
+
+##### Expectations of the Board
+
+- Maintain a transparent process in financial management of the foundation 
+- Responsible execution of fiscal duties to maintain and grow the foundation 
+- Hire an Executive Director who can execute on the expectations of the projects
+- Regularly review the performance of the Executive Director 
+- Review foundation performance with top-level project leadership quarterly 
+- Collaborate with representatives of the foundation on a yearly basis for accountability, multi-year strategy planning, and reviewing goal achievement 
+- Annual board meetup with members from top-level committee membership 
+
+##### Expectations of the Executive Director
+
+- Be an engaged participant of a/the projects' ecosystems. 
```

Comment:
This is an interesting one for me.  At times there has been pushback on having the ED helping to make decisions with the Node.js project.  At the same time if we want an engaged ED it would be great if they were an active member of the projects.  For example, if an existing TSC member was a good candidate for the ED, would we want them to pull back from the project if they got the position.

Reply:
I'm very -1 on the ED making decision for the projects. However I read "be engaged" in "know what's going on" at a high level.
I think there is a conflict of interest in the ED being part of the leadership of projects and being a collaborator on one for example. I think the ED cannot be in a leadership position because of conflict of interest, however they are welcomed to contribute if they like to - in their own time (they are not paid to work on the project).

Reply:
Yeah, I think the intention that was communicated to me with this one was that the ED needs to have the context of the communities and/or LISTEN very closely and trust the people who are providing them the context necessary to make decisions for the foundation. It helps to be in the know of these communities, but I do think there needs to be some lines drawn with what isn't appropriate to do if a person lands in the ED position that has leadership/collaborator responsibilities in one of the projects under the foundation scope.